src/sg/gov/tech/geekout/1.0/sample-document.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "GeekOut2020",
+  "recipient": {
+    "name": "Matthea Loo"
+  },
+  "programme": {
+    "name": "GeekOut 2020",
+    "startDate": "12-Oct-20",
+    "endDate": "14-Oct-20"
+  },
+  "signatory": {
+    "name": "Alice",
+    "position": "Boss",
+    "organisation": "ABC",
+    "signature": "signature"
+  },
+  "$template": {
```

Comment:
why adding $template ?

Reply:
Oh I thought that the raw document has to contain the template so after wrapping and being issued, when the document is dropped into the opencerts page it will display the right template.